### PR TITLE
Added Message from the President

### PR DESCRIPTION
**Pull request by samyak24jain**

### CODE REVIEW

File: src/templates/message.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %} {% block content %}
+<link href="/static/site/css/message.css" rel="stylesheet">
```
Comment (rohitvarkey) [inline src/templates/message.html]: Don't use these absolute links.  `{% static site/css/message.css}` should work I think.

File: src/templates/message.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+<div class="row">
+    <div class="col s2 card-image small presi-image-div">
+              <img class="presi-image" src="/static/site/img/presi.jpg">
```
Comment (rohitvarkey) [inline src/templates/message.html]: same as above

**Comment by rohitvarkey** [comment]
@samyak24jain Where are the Django parts? You added a corresponding urls and views no? Commit those and this should be ready for a merge.

**Comment by samyak24jain** [comment]
messagescreenshot1

**Comment by samyak24jain** [comment]
@rohitvarkey 
* Committed urls.py and views.py
* Changed the absolute urls to static
* Uploaded screenshot

**Comment by rohitvarkey** [comment]
Good stuff! :+1:

**Pull request merged**